tests/ert/unit_tests/scenarios/test_summary_response.py

```diff
@@ -236,3 +236,36 @@ def test_that_mismatched_responses_gives_nan_measured_data(ert_config, prior_ens
     assert pd.isna(fopr_2.loc[0].iloc[0])
     assert pd.isna(fopr_2.loc[1].iloc[0])
     assert pd.isna(fopr_1.loc[2].iloc[0])
+
+
+def test_reading_past_2263_is_ok(ert_config, storage, prior_ensemble):
```

Comment:
This should be squashed into the previous commit as the bug is fixed there, and so the test belongs along side that. Feel free to write a longer commit body of the first commit explaining the reason behind this change and the implications.